src/lib/Transport/index.ts

```diff
@@ -135,13 +149,27 @@ export default class Transport {
       ? (`${headers.referer}&context=${context}`)
       : (`${headers.referer}?userId=27&customer=dev&locale=en-US&platform=desktop&context=${context}`);
 
-    const newHeaders = {
+    if (this.isMultiPartRequest(headers)) {
+      // remove properties that are provided by the form-data library in request
+      return {
+        ...Object.keys(headers).reduce(
```

Comment:
Nice!

Reply:
haha, I was debating just using the `del` operator, but I knew that you would be upset if I made the headers mutable.